src/WebJobs.Extensions.ApiHub/Common/CodeFile2.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host.Bindings;
```

Comment:
Why is this file called CodeFile2? Also each interface needs to be in its own file - when you run fxcop/stylecop it should say so